addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1278,23 +1278,28 @@ registry.FullScreenHeight = publicWidget.Widget.extend({
      * @private
      */
     _adaptSize() {
-        const height = this._computeIdealHeight();
-        this.el.style.setProperty('min-height', `${height}px`, 'important');
+        const height = this._computeMainTopPosition();
```

Comment:
```suggestion
        // TODO made this way so custo of _computeIdealHeight still work, to adapt in master
        const height = $(window).outerHeight() - this._computeIdealHeight();
```

Reply:
ok, done, I did not think about that. for compatibility, I had stopped at "not deleting the method so customization or call to it would still work" not at having customization of the function still taken into account